core/object/configuration_info.h

```diff
@@ -59,6 +59,11 @@ class ConfigurationInfo {
 	StringName property_name;
 	Severity severity;
 
+	static HashSet<String> queued_errors_to_print;
```

Comment:
```suggestion
	inline static HashSet<String> queued_errors_to_print;
```
And remove from cpp.